src/ncp/ncp_openthread.cpp

```diff
@@ -265,6 +266,17 @@ otError ControllerOpenThread::ApplyFeatureFlagList(const FeatureFlagList &aFeatu
         error = SetOtbrAndOtLogLevel(otbrLogGetDefaultLevel());
     }
 
+#if OTBR_ENABLE_TREL
+    if (aFeatureFlagList.enable_trel())
+    {
+        otTrelEnable(mInstance);
```

Comment:
Is there any error code return from the otTrelEnable API? May need to return the error if any.

Reply:
`otTrelEnable` returns void